hw/bsp/nrf51dk/include/bsp/bsp.h

```diff
@@ -37,8 +37,44 @@ extern uint8_t _ram_start;
 #define RAM_SIZE        0x8000
 
 /* LED pins */
-#define LED_BLINK_PIN   (21)
-#define LED_2           (22)
+#define LED_1           (17)
+#define LED_2           (18)
+#define LED_3           (19)
+#define LED_4           (20)
+#define LED_BLINK_PIN   (LED_1)
```

Comment:
I did not cross verify everything with the schematic but LED_BLINK_PIN was 21 earlier and now it's getting changed to 17. Is this intentional ?